operations/companies.md

```diff
@@ -244,36 +326,196 @@ Adds a new company. Note this operation supports [Portfolio Access Tokens](../gu
+| `Id` | string | required | Unique identifier of the company. |
+| `ChainId` | string | required | Unique identifier of the chain. |
+| `Name` | string | required | Name of the company. |
+| `MotherCompanyId` | string | optional | Unique identifier of mother company. |
+| `InvoicingEmail` | string | optional | Email for issuing invoices to the company. |
+| `WebsiteUrl` | string | optional | The website url of the company. |
+| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
+| `Options` | [CompanyOptions](companies.md#companyoptions) | optional | Options of the company. |
+| `CreditRating` | [CreditRating](companies.md#creditrating) | optional | Credit rating to define creditworthiness of the company. |
 | `Department` | string | optional | The internal segmentation of a company, e.g. sales department. |
 | `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
-| `CreditRating` | [Credit rating](#credit-rating) | optional | Credit rating to define creditworthiness of the company. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the company from external system. |
 | `ReferenceIdentifier` | string | optional | External system identifier - custom identifier used by an external system such as an external database. |
-| `WebsiteUrl` | string | optional | The website url of the company. |
+| `AccountingCode` | string | optional | Accounting code of the company. |
+| `AdditionalTaxIdentifier` | string | optional | Additional tax identifier of the company. |
+| `BillingCode` | string | optional | Billing code of the company. |
+| `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
+| `ContactPerson` | string | optional | Contact person of the company. |
+| `ElectronicInvoiceIdentifier` | string | optional | Electronic invoice identifier of the company. |
+| `Identifier` | string | optional | Other identifier of the company, e.g. legal identifier. |
+| `Iata` | string | optional | Iata of the company. |
+| `IsActive` | boolean | required | Whether the company is still active. |
+| `Notes` | string | optional | Additional notes. |
+| `Number` | integer | required | Unique number of the company. |
+| `TaxIdentifier` | string | optional | Tax identification number of the company. |
+| `Telephone` | string | optional | Contact telephone number. |
+| `CreatedUtc` | string | optional | Date of [Company](companies.md#company) creation date and time. |
+| `UpdatedUtc` | string | optional | Date of [Company](companies.md#company) last update date and time. |
+| `AddressId` | string | optional | Unique identifier of the company [Address](addresses.md#account-address). |
+| `MergeTargetId` | string | optional |  |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of company from external system. |
+| `IsUpdatedByMe` | boolean | optional |  |
```

Comment:
❓ What description?

Reply:
Ooh, that's a new one, I think we have it still behind FF. Just hide it behind InternalImplementation. While it will be exposed in the response, it will be null for all partners.